app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.stories.tsx

```diff
@@ -35,81 +44,61 @@ export default {
   ],
 };
 
-export const ReviewNotRequested = () => <Subject />;
+const subjectWithProps = (
+  props?: React.ComponentProps<typeof Subject>,
+  storyName?: string,
+) => {
+  const story = () => <Subject {...props} />;
+  if (storyName) story.storyName = storyName;
+  return story;
+};
 
-export const ReviewRequestedUserCanReview = () => (
-  <Subject
-    {...{
-      reviewRequestEvent: mockChangelog(),
-      canReview: true,
-    }}
-  />
-);
+export const ReviewNotRequested = subjectWithProps({}, "Review not requested");
 
-export const ReviewPendingInRemoteSettingsUserCanReview = () => (
-  <Subject
-    {...{
-      reviewRequestEvent: mockChangelog(),
-      approvalEvent: mockChangelog("def@mozilla.com"),
-      canReview: true,
-    }}
-  />
-);
+export const ReviewRequestedUserCanReview = subjectWithProps({
+  ...reviewRequestedBaseProps,
+  canReview: true,
+});
 
-export const ReviewTimedOutInRemoteSettingsUserCanReview = () => (
-  <Subject
-    {...{
-      reviewRequestEvent: mockChangelog(),
-      approvalEvent: mockChangelog("def@mozilla.com"),
-      timeoutEvent: mockChangelog("ghi@mozilla.com"),
-      canReview: true,
-    }}
-  />
-);
+export const ReviewPendingInRemoteSettingsUserCanReview = subjectWithProps({
+  ...reviewPendingInRemoteSettingsBaseProps,
+  canReview: true,
+});
```

Comment:
Refactoring toward issue #5004 but also extracting some more mocks to share with tests.